Library/Homebrew/extend/ENV/std.rb

```diff
@@ -59,33 +52,18 @@ def setup_build_environment(formula = nil)
       gcc_formula = gcc_version_formula($&)
       append_path "PATH", gcc_formula.opt_bin.to_s
     end
+  end
+  alias generic_setup_build_environment setup_build_environment
```

Comment:
Could you change this and all other `alias` instances to use `alias_method` instead, as we've done in the past and is now codified in our RuboCop configuration as of #489?